src/components/Account/Account.scss

```diff
@@ -331,8 +331,8 @@ input {
 }
 
 .submitButton:disabled {
-  background-color: rgba(200, 200, 200, 0.5);
-  border: 1px solid rgba(200, 200, 200, 1);
+  background-color: $disabled-button-color;
```

Comment:
how this related to 3box ?

Reply:
https://github.com/daostack/alchemy/pull/1947#issuecomment-658451101